kubernetes/format.go

```diff
@@ -0,0 +1,52 @@
+package kubernetes
+
+import (
+	"github.com/flanksource/gomplate/v3/conv"
+	"github.com/flanksource/kubectl-neat/cmd"
+	"github.com/google/cel-go/cel"
+	"github.com/google/cel-go/common/types"
+	"github.com/google/cel-go/common/types/ref"
+)
+
+func Neat(in, outputFormat string) (string, error) {
+	out, err := cmd.NeatYAMLOrJSON([]byte(in), outputFormat)
+	return string(out), err
+}
+
+func k8sNeat() cel.EnvOption {
+	return cel.Function("k8s.neat",
+		cel.Overload("k8s_neat",
+			[]*cel.Type{cel.StringType},
```

Comment:
Can we support objects as well ?

Reply:
couldn't make that work due to this bug in serialization - https://github.com/flanksource/gomplate/issues/44

will work on that.